### PR TITLE
Cypress/E2E: Fixed known failed E2E tests

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fixed known failed E2E tests

#### Ticket Link
To follow

#### Screenshots
Screenshot 2023-06-12 at 7 42 29 PM
Screenshot 2023-06-12 at 7 43 05 PM
Screenshot 2023-06-12 at 7 43 41 PM
Screenshot 2023-06-12 at 7 44 04 PM
Screenshot 2023-06-12 at 7 44 27 PM
Screenshot 2023-06-12 at 7 45 04 PM

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e-tests/cypress/tests/integration/channels/auth_sso/authentication_2_spec.ts [hunk 1]
```diff
@@ -109,7 +109,7 @@ describe('Authentication', () => {
         // # Go to sign up with email page
         cy.visit('/signup_user_complete');
 
-        cy.get('#input_email', {timeout: TIMEOUTS.ONE_MIN}).type(`Hossein_Is_The_Best_PROGRAMMER${getRandomId()}@BestInTheWorld.com`);
+        cy.get('#input_email', {timeout: TIMEOUTS.ONE_MIN}).type(`test-${getRandomId()}@example.com`);
```
Comment (yasserfaraazkhan) [inline e2e-tests/cypress/tests/integration/channels/auth_sso/authentication_2_spec.ts:112]: 😄

**Review by yasserfaraazkhan — APPROVED** [review]

**Review by mvitale1989 — APPROVED** [review]
Thanks Saturn, lgtm!

**Pull request merged**